packages/Toggle/src/Toggle.js

```diff
@@ -377,6 +377,7 @@ export default class Toggle extends PureComponent {
     className: PropTypes.string,
     onChange: PropTypes.func,
     name: PropTypes.string,
+    id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
I personally think only permitting strings would be fine. Using numbers is a pretty bad practice with IDs anyways.

Reply:
I had the same thought, but in practice this may be more of a nuisance than necessary, as React does the conversion internally anyways.

Reply:
Should we accept both or just the string?